app/src/hooks.server.ts

```diff
@@ -2,13 +2,17 @@ import type { Handle, HandleServerError } from '@sveltejs/kit';
 import path from 'path';
 import { config as configDotenv } from 'dotenv';
 import { logger } from '$lib/utils/logger';
+import * as Sentry from '@sentry/node';
```

Comment:
Ça m'a pris un moment pour comprendre pourquoi on passait un objet `Sentry` à `initSentry` au lieu d'avoir l'import directement dans `lib/utils/sentry`. Maintenant que j'ai compris que c'était pour réutiliser `initSentry` aussi bien côté navigateur que serveur c'est plus clair. Peut-être qu'on pourrait clarifier ça en explicitant l'import ?
```suggestion
import * as NodeSentry from '@sentry/node';
```